packages/start/fs-router/router.js

```diff
@@ -386,17 +386,20 @@ export function stringifyAPIRoutes(
    * @return {string}
    */
   function _stringifyRoutes(/** @type {(RouteConfig)[]} */ routes) {
+    const methodNotAllowed = jsFile.addNamedImport("methodNotAllowed", "solid-start/api/utils");
```

Comment:
Could also just write `./utils` instead of `solid-start/api/utils`. Not sure if one is better than the other.